src/controller/CommissioningWindowOpener.cpp

```diff
@@ -175,23 +180,38 @@ CHIP_ERROR CommissioningWindowOpener::OpenCommissioningWindowInternal(Messaging:
 {
     ChipLogProgress(Controller, "OpenCommissioningWindow for device ID 0x" ChipLogFormatX64, ChipLogValueX64(mNodeId));
 
-    ClusterBase cluster(exchangeMgr, sessionHandle, mTargetEndpointId);
+    ClusterBase cluster(exchangeMgr, sessionHandle, mJointCommissioning ? 1 : mTargetEndpointId);
```

Comment:
I don't think this is correct as the Joint Fabric Administrator Cluster may stay on any endpoint (it's not mandatory to stay on Endpoint 1). Maybe one option would be to mandate the endpoint-id on the open-joint-commissioning-window command?

Reply:
Added new argument to set endpoint id